app/assets/src/components/CreateUser.jsx

```diff
@@ -280,6 +283,20 @@ class CreateUser extends React.Component {
                 />
                 <label htmlFor="user_name">Name</label>
               </div>
+              <div className="input-field">
+                <i className="fa fa-building" aria-hidden="true" />
+                <input
+                  type="text"
+                  onChange={e =>
```

Comment:
Nit: For consistency, can you make a `handleInstitutionChange` method?

Reply:
ty